de.tubs.variantsync.core/src/de/tubs/variantsync/core/VariantSyncComposer.java

```diff
@@ -90,4 +94,17 @@ public void copyNotComposedFiles(Configuration config, IFolder destination) {
 
 	}
 
+	@Override
+	public void buildPartialFeatureProjectAssets(IFolder sourceFolder, ArrayList<String> removedFeatures,
+			ArrayList<String> mandatoryFeatures) throws IOException, CoreException {
+		// TODO Auto-generated method stub
+		
+	}
+
+	@Override
+	public boolean supportsPartialFeatureProject() {
+		// TODO Auto-generated method stub
```

Comment:
Remove TODO, explain why this method always returns `false`. Instead of this pattern, I would prefer moving this method into an interface and resolving this problem on compile time.

Reply:
I don't exactly understand what you mean with moving the method into an interface and resolving this problem on compile time.
The method is required by the `ComposerExtensionClass` which is extended by this class (respectively the `IComposerExtensionBase` interface).
The method always returns `false` because no partial Feature Project is needed within VariantSync.